examples/pos-mainchain/src/app/app.ts

```diff
@@ -1,9 +1,22 @@
-import { Application, PartialApplicationConfig } from 'lisk-sdk';
+import {
+	Application,
+	FeeModule,
+	MainchainInteroperabilityModule,
+	PartialApplicationConfig,
+	TokenModule,
+	NFTModule,
+} from 'lisk-sdk';
 import { registerModules } from './modules';
 import { registerPlugins } from './plugins';
 
 export const getApplication = (config: PartialApplicationConfig): Application => {
 	const { app } = Application.defaultApplication(config, true);
+	const tokenModule = new TokenModule();
+	const nftModule = new NFTModule();
+	const feeModule = new FeeModule();
+	const interoperabilityModule = new MainchainInteroperabilityModule();
+	interoperabilityModule.registerInteroperableModule(nftModule);
+	nftModule.addDependencies(interoperabilityModule.method, feeModule.method, tokenModule.method);
```

Comment:
You can use `const { app, method } = Application.defaultApplication(config, true);` and use that instead of creating duplicate objects for token, fee and interoperability for adding methods to nft, the only problem is with registering interoperable module, as of now in the example app we can do it in a hacky way in the example app like below and resolve it in another issue https://github.com/LiskHQ/lisk-sdk/issues/8691
```ts
	const { app, method } = Application.defaultApplication(config, true);
	const nftModule = new NFTModule();
	const interoperabilityModule = app['_registeredModules'].find(mod => mod.name === 'interoperability')
	interoperabilityModule.registerInteroperableModule(nftModule);
	nftModule.addDependencies(method.interoperability, method.fee, method.token);
```